app/models/program.rb

```diff
@@ -199,4 +204,30 @@ def check_languages_format
     # We check if every language is a valid ISO 639-1 language
     errors.add(:languages, 'must be ISO 639-1 valid codes') unless languages_array.select{ |x| ISO_639.find(x).nil? }.empty?
   end
+
+  ##
+  # Check if schedule_interval is a divisor of 60 minutes
+  #
+  def schedule_interval_divisor_60
+    errors.add(:schedule_interval, 'must be a divisor of 60') if schedule_interval > 0 && 60 % schedule_interval > 0
+  end
+
+  ##
+  # Unschedule all the events which don't fit
+  #
+  def unschedule_unfit_events
```

Comment:
I am not pretty sure of what are unfit events for you? I mean, what events are you destroying in this method?


Because there are several options here. Deleting all the `event_schedules` for this conference would be one option. As I think it is not very clear what should happen with the events when you change the granularity of the schedule. @differentreality @eug3nix ideas?